tests/event-bus/e2e-tester/e2e-tester.go

```diff
@@ -36,10 +36,10 @@ const (
 	fail    = 1
 	retries = 20
 
-	sourceIDHeader         = "Ce-Source-Id"
-	eventTypeHeader        = "Ce-Event-Type"
-	eventTypeVersionHeader = "Ce-Event-Type-Version"
-	customHeader           = "Ce-X-Custom-Header"
+	sourceIDHeader         = "ce-source"
+	eventTypeHeader        = "ce-type"
+	eventTypeVersionHeader = "ce-eventtypeversion"
+	customHeader           = "ce-xcustomheader"
 
```

Comment:
Following are the missing headers:
`ce-specversion`
`ce-id`
`ce-time`
`content-type`
IMO we should test as per the contract as mentioned in [examples](https://github.com/kyma-project/examples/pull/129/commits/5c2ca1263022687f7273903c2fb2e1dc59f09a11)

Reply:
Thanks for the catch! 
Done.